x/stakeibc/keeper/community_pool.go

```diff
@@ -250,7 +255,7 @@ func (k Keeper) FundCommunityPool(ctx sdk.Context, hostZone types.HostZone, toke
 	_, err := k.SubmitTxs(ctx,
 		hostZone.ConnectionId,
 		msgs,
-		types.ICAAccountType_COMMUNITY_POOL_RETURN,
+		senderAccountType,
```

Comment:
Why do you need to make this parametrized?  The account which sends the funds back into the pool is known to be this ICA and the type is known as well right?

Reply:
yeah for the rebate we fund from the withdrawal account. For the CP liquid stake, we fund from the community pool return account